modules/getting-started-cli-login.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-cli.adoc
+
+:_content-type: PROCEDURE
+[id="getting-started-cli-login_{context}"]
+= Logging in to the CLI
+
+You can log in to the OpenShift CLI (`oc`) to access and manage your cluster.
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster.
+* You must have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Log into {product-title} from the CLI using your username and password or with an oauth token:
```

Comment:
* oauth -> OAuth
* Also, since it's a single step, should be an unnumbered bullet (*)